test/functional/test_framework/test_node.py

```diff
@@ -841,6 +844,10 @@ def bumpmocktime(self, seconds):
     def wait_until(self, test_function, timeout=60):
         return wait_until_helper_internal(test_function, timeout=timeout, timeout_factor=self.timeout_factor)
 
+    def tx_in_orphanage(self, tx: CTransaction) -> bool:
+        """Returns true if the transaction is in the orphanage."""
+        found = [o for o in self.getorphantxs(verbosity=1) if o["txid"] == tx.rehash() and o["wtxid"] == tx.getwtxid()]
+        return len(found) == 1
```

Comment:
The original thought/rationale was that the function pertains to the orphanage, which is part of the node (although a `node` argument to the function could enable obtaining the same information).
No file stands out as a perfect fit.  Do you feel `test_framework/mempool_util.py` would be reasonable? It seems appropriate since the orphanage and mempool both deal with unconfirmed transactions and are somewhat related.

Let me know and I can update either way.

Reply:
Ah, yeah, `mempool_util.py` looks like a better alternative than `blocktools.py`.

`test_node.py` feels more like the high-level P2P stuff to me at least + lower level dynamic RPC invocation.

Reply:
Moved to `mempool_util.py`